### DISCUSSION TITLE
...

### DISCUSSION THREAD

**Discussion by rraammiinn** [vlang/v#17238] [Questions and Answers]
can it really be as fast as c and when would you release first stable version ?

**Comment by JalonSolov** [comment]
From the home page:
```
As fast as C (V's main backend compiles to human readable C), with equivalent code.
V does introduce some overhead for safety (such as array bounds checking, GC free), but these features
can be disabled/bypassed when performance is more important.
```
The first "stable" version will be released when it is ready.  We don't do dates.

V is actually fairly stable already, depending on the features you use.  You'll have to try it and see if it is stable enough for you, or if you want to wait.
